### PR TITLE
Milestone 2: Send and Receive Data

### PR DESCRIPTION
**Pull request by fabgrel10**
In this PR:

- [x] Use Gitflow
- [x] Use Webpack
- [x] Write plain HTML markup with minimum styling
- [x] Use ES6 modules
- [x] Create a new game  by using the Microverse project API.
- [x] Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- [x] Implement the form "Submit" button (sending data to the API).
- [x] Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,34 @@
...
+        <div id="app">
+          <header id="header">
+            <h1 id="title">Leaderboard</h1>
+          </header>
+          <main id="main">
+            <aside class="aside-left">
+              <div id="aside-left__header">
+                <h2 class="subtitle">Recent Scores</h2>
+                <button class="button" id="refresh-scoreboard">Refresh</button>
+              </div>
```
Comment (Emmyn5600) [inline src/index.html:21]: - [ ] When the Refresh button is clicked, it should hit the GET API endpoint causing the scores list to change.

Please work on the refresh button.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @fabgrel10 

Great work on making the project requirements 👏 
You've done well implementing some of the project requirements, but there are still some which aren't addressed yet, please work on it and ask for another review.
      
 ## Required Changes ♻️

- _Check the comment under the review._

## Optional suggestions

- n/a

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/index.html (hunk 1 above)
Comment (fabgrel10) [reply src/index.html:21]: @Emmyn5600 That exactly what the refresh button does:

I created an event listener for the button:

```
  app.js

  refreshScoreboard.addEventListener('click', getScores);
```

and then I'm calling the function getScores to fetch the results from te api:
```
  api.js: 

  async function getScores() {
    const apiCall = await fetch(url);
    const response = await apiCall.json();
    renderScoreboard(response.result);
    addScoreToLocalStorage(response.result);
 }
```
Can you review again or should I request a second opinion review?

Comment (Emmyn5600) [reply src/index.html:21]: Hi @fabgrel10, I see you have implemented the event listener but on my end the posted data are not being displayed when I click on the refresh

Comment (fabgrel10) [reply src/index.html:21]: @Emmyn5600 is working fine on my side and also in the live demo link.

This is the test :data that I get directly from the API:
ksnip_20211215-150914

And then if hit the refresh button from the demo link or from my local environment I'm getting the same data. It's working fine.

Comment (Emmyn5600) [reply src/index.html:21]: Yeah I see, so try to display them on the screen when a user clicks on the refresh button can see the posted data

Comment (fabgrel10) [reply src/index.html:21]: Again, when the user hits the refresh button the data is gotten from the API and the print to screen from the renderScoreboard function within getScores. And requesting a second review right now.

**Comment by fabgrel10** [comment]
For the reviewer:

I commented out line 18 in app.js to avoid auto refreshing the scoreboard. Now the only way to do it is through the refresh button. The changed was needed to avoid misunderstandings.

@Emmyn5600

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi @fabgrel10, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by e71az — COMMENTED** [review]
Hello @fabgrel10 👋,

_This is an additional review as a result of the second opinion request._

I did check the project and everything seemed to be running just fine before the last change that you did, the project is approved so good luck in your next projects!

Keep rocking 🚀!

**Pull request merged**